### ISSUE TITLE
[Feature Request] New animation component - distort

### ISSUE THREAD

**Issue by Duseylicious** [encounterplus/encounterplus#241]
![Screen Shot 2020-07-05 at 12 28 56 PM](https://user-images.githubusercontent.com/63576494/86540574-4e4c3380-bebb-11ea-82cc-ff1a9ca09b15.png)

**Comment by Splinter714** [comment]
@Duseylicious identical screenshot as #240 which I assume is unintentional. Do you have any additional information about this?

**Comment by Duseylicious** [comment]
Yes, I messed up.  Here is the original post - not mine, I was posting for someone else who had a request. https://discord.com/channels/565243968858751027/634387773872275466/729126812323414026
